examples/adapt/image_segmentation/README.md

```diff
@@ -0,0 +1,194 @@
+# SegNeXt on cityscapes dataset using SSF 
+
+## Overview 
+
+This guide provides a walkthrough of applying SegNeXt for instance segmentation on the cityscapes dataset using SSF (Scaling and Shifting the deep Features). SSF enables parameter efficient fine-tuning by proposing that performace simillar to full fine-tuning can be achieved by only scaling and shifting the features of a deep neural network. 
+
+## Table on Contents
+ - [Introduction](#introduction)
+ - [Requirements](#requirements)
+ - [Installation](#installation)
+ - [Dataset](#dataset)
+ - [Configuration](#configuration)
+ - [Adapting the model](#adapting-the-model)
+ - [Conclusion](#conclusion)
+ 
+
+## Introduction
+
+In this example we will be finetuning SegNeXt large model on a city scapes dataset uisng a PEFT technique called SSF (Scale and Shift deep Features). SSF uses two linear layers to learn the scale and shift factors for deep features and hence uses very few trainable parameters. 
+
+## Requirements
+
+Before you begin, ensure that you have the following:
+- A GPU-enabled environment with CUDA support.
+- The Nyuntam repository cloned and set up as per the [Installation Guide](#installation).
+- Docker
+
+## Installation
+
+### Step 1: Clone the Nyuntam Repository
+
+Clone the repository and navigate to the `nyuntam` directory:
+```bash
+$ git clone https://github.com/nyunAI/nyuntam.git
+$ cd nyuntam
+```
+
+### Step 2: Set Up the workspace
+
+To setup the environment to run nyuntam_adapt, we will have to pull the nyuntam_adapt docker image. 
+
+```bash 
+$ docker pull nyunadmin/nyunzero_adapt:v0.1
+
+$ docker run -it -d --gpus all -v /dev/shm:/dev/shm -v $(pwd):/workspace --name nyuntam_adapt_docker --network=host nyunadmin/nyunzero_adapt:v0.1 bash 
+
+$ docker exec -it nyuntam_adapt_docker /bin/bash
+
+$ cd /workspace/nyuntam
+```
+
+## Dataset 
+
+For this experiment we will be using the [CityScapes](https://www.kaggle.com/datasets/ipythonx/stanford-background-dataset) dataset. In this dataset we will only be using the "regions" as the segmentation maps. The data folder structure is formatted in the folowing way: 
```

Comment:
Do we have any formatting/modifications on this dataset to account only for the "regions" as segmentation? If yes, we can add details on the formatting step(s) too.